### ISSUE TITLE
Samsung VR not syncing

### ISSUE THREAD

**Issue by sboombastic3** [FredTungsten/ScriptPlayer#114]
Can't download the previous version of Samsung VR anymore.

**Comment by FredTungsten** [comment]
Duplicate of #113 

Well I'm not Samsung so there is nothing I can do about that.
However, according to a link in in the other issue, there is an apk mirror that has a working version:
https://www.apkmirror.com/apk/samsung-electronics-co-ltd/samsung-vr-2/samsung-vr-2-2-04-2-release/samsung-vr-2-04-2-android-apk-download/

**Comment by sboombastic3** [comment]
Thanks, yeah I know your not Samsung.  Lol.  I spoke to their developers
and they are pretty clueless to the fact they even have a presentation mode
so it won't be fixed anytime in the future.  I was hoping somebody found
some sort of work around.  I'm a network engineer guy so development is a
totally different game to me.  I did use the Old APK but Samsung updated it
after a week or so of everything working and then made it impossible to
upload it again.  I receive a can't install error.  I reached out to the
Samsung developers again and they could careless even after saying I have a
business that depends on me writing scripts.  Why a company feels they can
dictate how we utilize out $1000+ phones is beyond me....
